org.sofproject.topo.ui/META-INF/MANIFEST.MF

```diff
@@ -22,7 +22,8 @@ Import-Package: com.google.common.collect;version="21.0.0",
  com.google.inject;version="1.3.0",
  com.google.inject.binder;version="1.3.0",
  com.google.inject.multibindings;version="1.3.0",
- com.google.inject.util;version="1.3.0"
+ com.google.inject.util;version="1.3.0",
+ org.sofproject.gst.json
```

Comment:
Do not create a dependency from a common base to a specific single topology plug-in. If json serialization is supposed to be a common functionality, add it to this topo.ui plugin.

Reply:
ok, done